### PR TITLE
ETL unknown applicant 

### PR DESCRIPTION
**Pull request by lstod**
- remove ETL unknown applicant update

### CODE REVIEW

File: bin/migrate-oats-data/applications/migrate_application.py [hunk 1]
```diff
@@ -69,7 +69,7 @@ def process_application_etl(batch_size):
     update_application_incomplete_date(batch_size)
     process_application_parcels(batch_size)
     process_application_owners(batch_size)
-    process_application_applicant_on_submissions(batch_size)
+    # process_application_applicant_on_submissions(batch_size)
```
Comment (trslater) [inline bin/migrate-oats-data/applications/migrate_application.py:72]: Why comments and not deletions?

Comment (lstod) [reply bin/migrate-oats-data/applications/migrate_application.py:72]: currently working on debugging test, mekhti should have a separate MR addressing it soon

**Review by mhuseinov — APPROVED** [review]
👍

**Review by trslater — APPROVED** [review]
LGTM 🥇

**Pull request merged**